test/reactionsystem_core/events.jl

```diff
@@ -362,9 +362,9 @@ let
     sol = solve(jprob, SSAStepper(); seed)
     
     # Checks that all `e` parameters have been updated properly.
-    # Note that periodic discrete events are currently broken for jump processes.
+    # Note that periodic discrete events are currently broken for jump processes (and unlikely to be fixed soon due to have events are implemented).
```

Comment:
Do we have other event tests with jumps that check accuracy and which don't use periodic callbacks? (So aren't disabled?)

Reply:
The test got code for all three types of discrte events, but only the periodic one is actually disabled. The other two types are still tested:
```julia
# Tests that events are properly triggered for Jump simulations.
# Tests for all three types of discrete events.
let
    # Creates model with all types of events. The `e` parameters track whether events are triggered.
    rn = @reaction_network begin
        @parameters e1=0 e2=0 e3=0
        @discrete_events begin
            [1.0] => [e1 ~ 1]
            # 1.0 => [e2 ~ 1]
            (X > 1000.0) & (e3==0) => [e3 ~ 1]
        end
        (p,d), 0 <--> X
    end
    
    # Simulates the model for conditions where it *definitely* will cross `X = 1000.0`
    u0 = [:X => 999]
    ps = [:p => 10.0, :d => 0.001]
    dprob = DiscreteProblem(rn, u0, (0.0, 2.0), ps)
    jprob = JumpProblem(rn, dprob, Direct(); rng)
    sol = solve(jprob, SSAStepper(); seed)
    
    # Checks that all `e` parameters have been updated properly.
    # Note that periodic discrete events are currently broken for jump processes.
    @test sol.ps[:e1] == 1
    @test_broken sol.ps[:e2] == 1
    @test sol.ps[:e3] == 1
end
```

Reply:
Great.